docs/user-guide/admin-cli.md

```diff
@@ -653,17 +701,144 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy rm --name=some-policy
 Successfully removed policy: some-policy
 ```
+
+## Config
+
+Creates, removes, moves and reports on status of config DB. Config DB keeps authentication information, and without initializing it no access control is supported. Also, configuring access control will fail, e.g.
```

Comment:
Is the auth DB the same as the config DB? Can the terms be defined ahead and if these are the same then a single term is used in all places.